tests/integration/test_assets.py

```diff
@@ -62,8 +62,7 @@ def test_full_flow():
     # The transactions in an Asset Report with Insights should have a non-null
     # `name` (when available).
     assert (
-        report['items'][0]['accounts'][0]['transactions'][0]['name']
-    ) is not None
+        name_exists_for_some_some_transaction(report)) is True
```

Comment:
`is True` is redundant, `assert name_exists_for_some_some_transaction(report)` does the same.

Reply:
⚡️